nuxeo-ai-model/src/main/java/org/nuxeo/ai/bulk/DataSetUploadComputation.java

```diff
@@ -65,10 +73,15 @@ public void processRecord(ComputationContext context, String inputStreamName, Re
                                 if (document != null) {
                                     CloudClient client = Framework.getService(CloudClient.class);
                                     if (client.isAvailable()) {
-                                        client.uploadDataset(document);
+                                        log.info(String.format("Uploading dataset to cloud for command %s", cmd.getId()));
```

Comment:
You could use log4j2 loggers to avoid `format` computation if log level is not info.

Reply:
OK, just because you asked so nicely.